doc/lib/attributes.py

```diff
@@ -71,9 +71,9 @@ def _type_name(self, type_):
         if isinstance(type_, typing._GenericAlias):
             if type_.__origin__ is dict:
                 key_type, val_type = type_.__args__
-                return type_._name + f'[{self._type_name(key_type)}, {self._type_name(val_type)}]'
+                return 'Dict' + f'[{self._type_name(key_type)}, {self._type_name(val_type)}]'
             elif type_.__origin__ in (list, set):
-                return type_._name + f'[{self._type_name(type_.__args__[0])}]'
+                return 'List' + f'[{self._type_name(type_.__args__[0])}]'
```

Comment:
nit: why not include `Dict` and `List` into the formatstr-literal? also: how about `List of`? But I am also ok with your less verbose variant